app/src/main/res/layout/language_picker_bottom_sheet.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:paddingStart="@dimen/base_margin_triple"
+    android:paddingTop="@dimen/base_margin"
+    android:paddingEnd="@dimen/base_margin_triple"
+    android:paddingBottom="@dimen/base_margin_triple">
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:contentDescription="@null"
+        android:src="@drawable/bottom_sheet_handle" />
+
+    <TextView
+        android:id="@+id/header"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/base_margin_triple"
+        android:textAppearance="@style/Hedvig.TextAppearance.Headline5"
+        tools:text="Header" />
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginVertical="@dimen/base_margin"
+        android:text="@string/language_picker_modal.text"
+        android:textAppearance="@style/Hedvig.TextAppearance.Body1" />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recycler"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```

Comment:
We should explore whether we can make this `RecyclerView` not be `wrap_content` in height.

Reply:
It looks the same but i dont see why would want `match_parent`

Reply:
No recycling can occur when a `RecyclerView` is `wrap_content`